workspaces/remix-electron/src/index.mts

```diff
@@ -38,18 +39,29 @@ export async function initRemix({
 	mode,
 	publicFolder: publicFolderOption = "public",
 	getLoadContext,
+	esm = typeof require === "undefined",
 }: InitRemixOptions): Promise<string> {
-	const appRoot = app.getAppPath()
```

Comment:
this is the electron app root not the remix project's and therefore `publicFolder` will be wrong when resolved below, so I replaced it with `process.cwd()`